tests/test_open.py

```diff
@@ -21,6 +23,7 @@ def test_open_filename_with_exclamation(data_dir):
     assert fiona.open(path), "Failed to open !test.geojson"
 
 
+@requires_gdal21
```

Comment:
Tests were passing without this before, no?

Reply:
shrug, [maybe](https://github.com/Toblerity/Fiona/runs/2713722384)

Reply:
Oh, this change is something @mwtoews added to make things greeen, sorry, I forgot what I did here :)

Reply:
This is from #1037